### ISSUE TITLE
The loss function in model.py is different from the paper

### ISSUE THREAD

**Issue by xuejy19** [megvii-research/CVPR2023-DMVFN#6]
No matter whether a pre-trained model is used or not, I am unable to reproduce the results of the paper on UCF101. However, I found that there is some inconsistency between the loss function on line 67 of the model/model.py and the paper, and an additional loss_vgg term has been added. I am not sure if it is this loss term that is causing me to be unable to reproduce the results on UCF101. Could you provide more implementation details in UCF101? Thank you very much.

**Comment by huxiaotaostasy** [comment]
Adding vgg loss will reduce lpips (improve the continuity of the video), but it will also reduce psnr and ssim, and the training will become unstable. In fact, our model can achieve SOTA regardless of whether vgg loss is used or not. You can see if you have the same problem on cityscapes and KITTI.
